tests/helm_install.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+readonly ROOT_DIR="$(dirname "$(dirname "${0}")")"
+
+helm install test-openshift sumologic/sumologic \
```

Comment:
Note: this command is not changes at this moment, it is only moved from `tests/deploy_helm_chart.sh`